qubes/config.py

```diff
@@ -108,3 +108,6 @@
 
 #: profiles for admin.backup.* calls
 backup_profile_dir = '/etc/qubes/backup'
+
+#: site-local prefix for all VMs
+qubes_ipv6_prefix = 'fd09:24ef:4179:0000'
```

Comment:
Where did this come from? (I mean, after `fd09`).

Reply:
Randomly generated, as required by https://tools.ietf.org/html/rfc4193#section-3.2. The same for all Qubes instances, to not allow specific instance fingerprinting based on this.